script/1_0_2current_en_script/Arcueid/Day 7/07_00_ARC07_1B.txt

```diff
@@ -85,78 +85,78 @@ I turn around at the call of my name.
 "If you're feeling well, then do your best starting today. It's in your best interest to not take any more days off ♡"
 The returnee from France gives me something along the lines of a wink and heads toward the staff room.
 <Page25>
-I thought I'd be the first to arrive, but there was already several of my classmates in the classroom.
+I thought I'd be the first to arrive, but several of my classmates were already in the classroom.
 "You're early, Tohno. What happened on Friday?"
 "Morning. I was sick on Friday. Are you done with your morning practice?"
 <Page26>
-"Yep. Not only do we have to do after-school activities, but we also have to go far as to respect morning practice. We've only got a new team captain just recently and he's already a pain in the ass. Don't you think?"
+"Yep. Not only for after-school activities, but we're also being told to take care of ourselves even until morning practice. We've just gotten a new team captain recently and he's already a pain in the ass. Don't you think?"//「そーそー。放課後の部活動だけじゃなく、朝練まで自重しろって話になってきてさー。主将を交代したばっかだってのに、こんなんじゃ統率とれねぇよ。なぁ？」
```

Comment:
This is a positive change but the first clause should be "It's not just after-school activities," or something to that effect.